plugins/scaffolder/alpha-api-report.md

```diff
@@ -42,6 +42,7 @@ export type NextRouterProps = {
     actions?: boolean;
     tasks?: boolean;
   };
+  customFilters?: Array<React_2.ReactElement>;
```

Comment:
@benjdlambert thoughts on this? Does the new scaffolder page support the customization pattern from the catalog so that we can avoid this?

Reply:
So it's similar, but it's not as easy to wire components to together. You can provide your own `TemplateListPage` in which you can use the components out of `@backstage/plugin-scaffolder-react` to build your own and wire the customizations together but we've not spent time making a nice API for this, was kind of waiting for maybe some movement on #16876 to be honest rather than adding more props.

Reply:
Alright, so perhaps hold off on the prop change here for now then since there is an escape hatch

Reply:
I took a look at #16876 and it makes sense to make the entire scaffolder component adaptable and more customizable. I can hold off on this prop change for now. @Rugvip I'm curious to know when can we expect the RFC and wanted to know if our team can be of any help.

I also noticed another prop `templateFilters` to filter the templates based on the entity. I wonder if it is possible to use the property to filter based on the other personalized metadata?